src/index.jsx

```diff
@@ -20,17 +20,17 @@ class Tiers extends React.Component {
   }
 
   getBundleData() {
-    let bundleId = this.bundleId;
+    let bundleId = this.state.bundleId;
     $.ajax({
       method: 'GET',
       // url: '/tiersIncluded/:' + bundleId,
       url: '/tiersIncluded',
       datatype: 'json',
       success: (data) => {
         this.setState({
-        tier1Id: data[0].tierId,
-        tier2Id: data[1].tierId,
-        tier3Id: data[2].tierId
+          tier1Id: data[0].tierId,
+          tier2Id: data[1].tierId,
+          tier3Id: data[2].tierId
```

Comment:
I believe it's recommended that we use trailing commas after the last line of multi-line object and array expressions.